pypeit/core/findobj_skymask.py

```diff
@@ -124,7 +124,18 @@ def create_skymask(sobjs, thismask, slit_left, slit_righ, box_rad_pix=None, trim
     # DP: I think skymask should always be skymask_objflux & skymask_fwhm (i.e., not only when box_rad_pix is not None).
     # In the case of skymask_objflux | skymask_fwhm, if skymask_objflux cannot be computed, the entire slit
     # is used for sky calculation (i.e., skymask_fwhm will not have effect).
-    skymask = skymask_objflux & skymask_fwhm
+
+    # DP's change which I don't think we should adopt at this time.
```

Comment:
This was part of `objfind` for a long time.

Be careful here..

Reply:
Yes, I'm trying to restore the original behavior since this change in masking has some catastrophic consequences in some cases.